main.py

```
@@ -14,31 +15,67 @@ def order_menu():
 
-    while (user_input := input(menu)) != "4":
+    while (user_input := input(menu)) != "5":
         if user_input == "1":
-            DateOrdered = input("Input the date the order was placed: (mm-dd-yy)")
-            ManufacturerID = input("What is the manufacturer id? ")
-
-            database.create_order_and_delivery(DateOrdered, ManufacturerID)
-
+            try:
+                create_order()
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)